protocol/x/govplus/types/keys.go

```diff
@@ -6,5 +6,5 @@ const (
 	ModuleName = "govplus"
 
 	// StoreKey defines the primary module store key
-	StoreKey = ModuleName
+	StoreKey = "dydxgovplus"
```

Comment:
Store keys can't be prefixes of another apparently, so I can't use govplus with gov.

Reply:
Looks likes break the convention that `StoreKey` = `ModuleName`. Should we just update module name to `dydxgovplus` (and update directory names accordingly)?

Reply:
May be worth adding a comment here to reflect that

Reply:
Feel like it's not a particularly worthwhile convention to keep. For example not all cosmos modules even follow the convention: https://github.com/dydxprotocol/cosmos-sdk/blob/dydx-fork-v0.50.3/x/auth/types/keys.go#L12

I'd rather have a nicer module name than try to keep everything the same.